docs/user_guide/configuring.rst

```diff
@@ -19,6 +19,15 @@ doc path's _static folder, and use the following configuration:
 
    html_logo = "_static/logo.png"
 
+The logo links to ``master_doc`` by default. If you'd like to link to other page
```

Comment:
```suggestion
The logo links to the first page of your documentation by default. If you'd like to link to another page
```

Reply:
Agree on "other page" to "another page" but the logo really links to ``master_doc`` by default though and sometimes ``master_doc`` is not the landing page (e.g. numpy dev docs).

Reply:
hmmm - fair enough, what do you think about the new language suggestion? The thing is that most users don't know what `master_doc` is so I prefer using terminology that the "average" user will likely understand

Reply:
another thing we could do instead is include a link to the Sphinx documentation for `master_doc`

Reply:
I think the new language suggestion is pretty informative!